src/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -145,4 +150,27 @@ protected function filterProperty(string $property, $value, QueryBuilder $queryB
             ->andWhere(\sprintf('%s.%s = :%s', $alias, $field, $valueParameter))
             ->setParameter($valueParameter, $value);
     }
+
+    /**
+     * @return array<string, string>
+     */
+    public function getSchema(Parameter $parameter): array
+    {
+        return $parameter->getSchema() ?? ['type' => 'boolean'];
+    }
+
+    public function getOpenApiParameters(Parameter $parameter): OpenApiParameter|array|null
+    {
+        $in = $parameter instanceof QueryParameter ? 'query' : 'header';
+        $key = $parameter->getKey();
+
+        return [
+            new OpenApiParameter(
+                name: $key,
+                in: $in,
+                required: false,
+                schema: ['type' => 'boolean'],
+            ),
+        ];
```

Comment:
same as for the ODM filter

Reply:
What does that mean? In what way should I remove it?